types/index.d.ts

```diff
@@ -0,0 +1,461 @@
+// TypeScript Version: 3.5
+import * as React from 'react';
+import * as Client from 'stream-chat';
+import SeamlessImmutable from 'seamless-immutable';
+
+export interface ChatProps {
+  client: Client.StreamChat;
+  theme?: Theme;
+}
+
+export type Theme = 'messaging light' | 'messaging dark';
```

Comment:
users of this library can come up with their own theme, in that case it will be something else than 'messaging light' or 'messaging dark'